### ISSUE TITLE
add icon for pizza only fast food

### ISSUE THREAD

**Issue by RedAuburn** [organicmaps/organicmaps#6239]
from OSM World discord:

>Casper:
Maybe cuisine=pizza can have its own icon, especially as a replacement for the burger icon for amenity=fast_food places.
And it's a super popular tag: https://taginfo.openstreetmap.org/keys/cuisine#values

might not be possible yet because of the way cuisines are handled? @pastk

**Comment by pastk** [comment]
`amenity|fast_food|pizza;[amenity=fast_food][cuisine=pizza]` should work, but it'll match POIs tagged as e.g. `cuisine=burger;pizza;sandwich` which we don't want I think (pre-processing is osm2type.cpp splits all colon-separated cuisines into separate tags).

If we want to match `cuisine=pizza` places only then we'd need to add some exception to this pre-processing.

**Comment by pastk** [comment]
Yeap, a more cuisine-neutral fast food icon would be great!
Would be good to make it different from restraunts and cafes still.
Any other ideas?
@dvdmrtnz

**Comment by bveldkamp** [comment]
Knife and fork for a fastfood place would seem... odd. But I must agree, it's better than the burger icon we have now

**Comment by biodranik** [comment]
A better icon is one that is recognizable by most users, not the one that better reflects reality.

**Comment by dvdmrtnz** [comment]
> A better icon is one that is recognizable by most users, not the one that better reflects reality.

I agree. It's like the save icon being a floppy disk: https://uxdesign.cc/the-floppy-disk-save-icon-visual-language-of-an-era-long-gone-93f74efc9f9

I would not change the restaurant icon. Fork and knife is very recognizable as a restaurant. Maybe we could change the fast-food icon but I think a burger is very recognizable as fast-food, even if it doesn't match every single fast-food place. I'm not sure what else to use. 🤔

**Comment by bveldkamp** [comment]
Both Google and Bing use knife+fork for everything, fastfood *and* sit down, so perhaps that's not such a bad idea after all. 

If we're going to change it, there are some obvious icons for specific cuisines, e.g. a pizza slice, a burrito, a piece of sushi, a slice of bread, ... so maybe we could add those, and use knife+fork (or even the burger) for all others? Otoh it may become too messy with all different kinds of icon.

**Comment by 501Ghost** [comment]
> Both Google and Bing use knife+fork for everything, fastfood and sit down

Magic Earth even uses a fork+knife icon for `amenity=bbq`, which wouldn't be my choice but it illustrates how common the icon is in other places.

**Comment by theadventurer62** [comment]
Restarting this discussion, I made an idea post about this: https://github.com/orgs/organicmaps/discussions/10952.

We could probably use Maki icons, as they're compatible with the project and are already used by iD.
